src/app/directives/documentation.js

```diff
@@ -231,6 +232,29 @@
               .velocity('slideUp');
           }
         };
+
+        $scope.getBeautifiedExample = function (value) {
```

Comment:
`api-console.js` is a generated file that concatenates all the js files in the project